builders/interoperability/xcm/xc-registration/xc-integration.md

```diff
@@ -0,0 +1,320 @@
+
+The accepted values for the relay chain are `polkadot` (default), `kusama`, and `moonbase`.
+
+For example, Moonbase Alpha's sovereign account for both the relay chain and other parachains can be obtained with the following:
+
+```bash
+yarn calculate-sovereign-account --p 1000 --r moonbase
+```
+
+Which should result in the following response:  
+
+```text
+Sovereign Account Address on Relay: 0x70617261e8030000000000000000000000000000000000000000000000000000
+Sovereign Account Address on other Parachains (Generic): 0x7369626ce8030000000000000000000000000000000000000000000000000000
+Sovereign Account Address on Moonbase Alpha: 0x7369626ce8030000000000000000000000000000
+```
+
+## Moonriver & Moonbeam XCM Integration Overview {: #moonriver-moonbeam }
+
+From a technical perspective, the process of creating an HRMP channel with Moonriver and Moonbeam is nearly identical. However, engagement with the Moonbeam community is crucial and required before a proposal will pass.
+
+Please check the HRMP channel guidelines that the community voted on for [Moonriver](https://moonriver.polkassembly.network/referenda/0){target=_blank} and [Moonbeam](https://moonbeam.polkassembly.network/proposal/21){target=_blank} before starting.
+
+The process can be summarized in the following steps:
+
+1. Open (or ensure there is) an HRMP channel from your chain to Moonriver/Moonbeam. Optionally, register MOVR/GLMR
+2. Create [two Moonbeam Community forum posts](#forum-templates) with some key information for the XCM integration:
+    - An [XCM Disclosure post](/builders/interoperability/xcm/xc-registration/forum-templates#xcm-disclosure), where you'll provide some disclosures about the project, the code base, and social network channels
+    - An [XCM Proposal post](/builders/interoperability/xcm/xc-registration/forum-templates##xcm-proposals), where you'll provide some technical information about the proposal itself
+3. Create a batch proposal on Moonbeam/Moonriver to:
+    1. Accept the incoming HRMP channel
+    2. Propose the opening of an outgoing HRMP channel from Moonriver/Moonbeam
+    3. Register the asset as an [XC-20 token](/builders/interoperability/xcm/xc20/overview){target=_blank} (if applicable)
+
+      The normal enactment times are as follows:  
+
+      - **Moonriver** - proposals should be done in the General Admin Track from [OpenGov](/learn/features/governance/#opengov){target=_blank}, in which the Decision Period is approximately {{ networks.moonriver.governance.tracks.general_admin.decision_period.time }} and the enactment time is at least {{ networks.moonriver.governance.tracks.general_admin.min_enactment_period.time }}
+      - **Moonbeam** - approximately a {{ networks.moonbeam.democracy.vote_period.days }}-day Voting Period plus a {{ networks.moonbeam.democracy.enact_period.days }}-day enactment time
```

Comment:
If this gets merged before RT240x gets rolled out to Moonbeam, I'll need to adjust this. Otherwise, I can update it in https://github.com/moonbeam-foundation/moonbeam-docs/pull/694

Reply:
Same as other comment